lib/Traits/CatalogMemberTraits.ts

```diff
@@ -47,6 +47,13 @@ export class InfoSectionTraits extends ModelTraits {
   })
   contentAsObject?: JsonObject;
 
+  @primitiveTrait({
+    type: "boolean",
+    name: "Show",
+    description: "Indicates if this info section showing (not collapsed)."
+  })
+  show? = true;
```

Comment:
Rename this `collapsed`, maybe?  I know you've got the description there explaining otherwise, but `show` might make it seem like it won't be shown at all, as opposed to collapsed.

Reply:
I agree that `collapsed` would be better, but I am following the pattern used for `ShortReportTraits` which uses `show` in the same way